src/pybind/mgr/nfs/module.py

```diff
@@ -24,82 +25,104 @@ def __init__(self, *args, **kwargs):
             self.inited = True
 
     @CLICommand('nfs export create cephfs', perm='rw')
-    def _cmd_nfs_export_create_cephfs(self,
-                                      fsname: str,
-                                      clusterid: str,
-                                      binding: str,
-                                      path: str = '/',
-                                      readonly: bool = False) -> Tuple[int, str, str]:
+    def _cmd_nfs_export_create_cephfs(
+            self,
+            fsname: str,
+            cluster_id: str,
+            binding: str,
+            path: Optional[str] = '/',
+            readonly: Optional[bool] = False,
+            addr: Optional[List[str]] = None,
```

Comment:
```suggestion
            client_addr: Optional[List[str]] = None,
```

can potentially be confused with the ganesha server ip addrs?